test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -5,7 +5,7 @@ Import-Module HelpersCommon
 
 # Test Settings:
 # This is the list of PowerShell modules for which we test update-help
-$powershellCoreModules = @(
+[string[]] $powershellCoreModules = @(
```

Comment:
Please revert the unrelated change. Below too.

I understand that you have your own scripting style, however a general practice is to follow the style of the file and the traditions of the project.
Also please do not mix functional and styling changes and unrelated changes in one PR.

Reply:
If I fix only **UserScope**, the tests will fail because they will actually run.  The only reason they have not failed yet is that they were effectively disabled.

Reply:
I agree with @yecril71pl, this is fine and looks like it was necessary.